modules/installation-generate-aws-user-infra-install-config.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+
+ifeval::["{context}" == "installing-restricted-networks-aws"]
+:restricted:
+endif::[]
+
+[id="installation-generate-aws-user-infra-install-config_{context}"]
+= Creating the installation configuration file
+
+Generate and customize the installation configuration file that the
+installation program needs to deploy your cluster.
+
+.Prerequisites
+
+* Obtain the {product-title} installation program and the pull secret for your
+cluster.
+ifdef::restricted[]
+For a restricted network installation, these files are on your bastion host.
+endif::restricted[]
+
+.Procedure
+
+. Obtain the `install-config.yaml` file.
+.. Run the following command:
++
+----
+$ ./openshift-install create install-config --dir=<installation_directory> <1>
+----
+<1> For `<installation_directory>`, specify the directory name to store the
+files that the installation program creates.
++
+[IMPORTANT]
+====
+Specify an empty directory. Some installation assets, like bootstrap X.509
+certificates have short expiration intervals, so you must not reuse an
+installation directory. If you want to reuse individual files from another
+cluster installation, you can copy them into your directory. However, the file
+names for the installation assets might change between releases. Use caution
+when copying installation files from an earlier {product-title} version.
+====
+.. At the prompts, provide the configuration details for your cloud:
+... Optional: Select an SSH key to use to access your cluster machines.
++
+[NOTE]
+====
+For production {product-title} clusters on which you want to perform installation
+debugging or disaster recovery on, you must provide an SSH key that your `ssh-agent`
+process uses to the installation program.
+====
+... Select *aws* as the platform to target.
+... If you do not have an AWS profile stored on your computer, enter the AWS
+access key ID and secret access key for the user that you configured to run the
+installation program.
+... Select the AWS region to deploy the cluster to.
+... Select the base domain for the Route53 service that you configured for your cluster.
+... Enter a descriptive name for your cluster.
+... Paste the pull secret that you obtained from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page.
+
+. Edit the `install-config.yaml` file to set the number of compute, or worker,
```

Comment:
@wking correct me if I'm wrong, but this section should be removed due to https://bugzilla.redhat.com/show_bug.cgi?id=1755073.

Reply:
I think so, but prob wait until https://github.com/openshift/installer/pull/2402 lands or is otherwise sorted.